s3/src/main/resources/reference.conf

```diff
@@ -89,4 +89,20 @@ alpakka.s3 {
   # This flag may disable the validation.
   # See https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingMetadata.html
   validate-object-key = true
+
+  # Settings corresponding to automatic retry of chunks in a streamed multipart upload.
+  retry-settings {
```

Comment:
It makes sense to introduce `multipart-upload.retry-settings` in case we discover more needs for retry configurations.

It could rely on config inheritance:
```
multipart-upload.retry-settings = ${alpakka.s3.retry-settings}
```

Reply:
Done.